plugin/src/main/java/com/denizenscript/denizen/objects/properties/PropertyRegistry.java

```diff
@@ -106,6 +106,7 @@ public static void registerMainProperties() {
         PropertyParser.registerProperty(EntityJumpStrength.class, EntityTag.class);
         PropertyParser.registerProperty(EntityKnockback.class, EntityTag.class);
         PropertyParser.registerProperty(EntityMarker.class, EntityTag.class);
+        PropertyParser.registerProperty(EntityMaterial.class, EntityTag.class);

```

Comment:
Was discussed on discord iirc - this has nothing to do with the other parts of this same PR